### PR TITLE
Fix/timer style

### PR DESCRIPTION
**Pull request by tylrs**
# Description

Add completed style section, fixed timer spacing, adjust font-sizes and colors, & added log activity button.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality

### CODE REVIEW

File: Activity.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ class Activity {
   }
 
   countdown() {
-
+      console.log("Toad Noises");
```
Comment (hoomberto) [inline Activity.js:12]: What is this?!

File: styles.css [hunk 2]
```diff
@@ -228,7 +229,7 @@ input {
   display: flex;
   flex-direction: column;
   background-color: #46424D;
-  height: 85%;
+  height: 80%;
```
Comment (hoomberto) [inline styles.css:235]: Hopefully this makes the section sizing look juuuust right

File: index.html [hunk 3]
```diff
@@ -14,10 +15,12 @@
       <section class="current-activity hidden">
         <h2>Current Activity</h2>
         <section class="timer-container" id="actual-timer-container">
-          <p>Deep Breathing</p>
-          <section class="timer-numbers">00:00</section>
```
Comment (AshleyOh-bit) [inline index.html:18]: good catch - this would have messed up our styling

File: styles.css [hunk 4]
```diff
@@ -225,21 +225,23 @@ input {
 }
 
 #actual-timer-container {
-  align-items: center;
   display: flex;
   flex-direction: column;
   background-color: #46424D;
-  height: 68%;
```
Comment (AshleyOh-bit) [inline styles.css:232]: This is a big change - I hope it resolved the issues we were having with the bar at the bottom of the section.

**Review by AshleyOh-bit — APPROVED** [review]

**Pull request merged**